ReleaseHistory.md

```diff
@@ -8,7 +8,7 @@
 
 ### API Changes
 
-* Added `NativeWindowHandle` wrapper and replaced `pNativeWndHandle` and `pDisplay` members with it in `EngineGLCreateInfo` (API Version 2400XX)
+* Added `NativeWindow` wrapper and replaced `pNativeWndHandle` and `pDisplay` members with it in `EngineGLCreateInfo` (API Version 2400XX)
```

Comment:
Please update this to 240053, as well as [here](https://github.com/DiligentGraphics/DiligentCore/blob/master/Graphics/GraphicsEngine/interface/APIInfo.h#L33). I also updated the file with C interface that I forgot to mentiond.